umm-lib/test/cmr/umm/test/echo10/related_url.clj

```diff
@@ -0,0 +1,25 @@
+(ns cmr.umm.test.echo10.related-url
+  "Tests functions for granule related urls"
+  (:require [clojure.test :refer :all]
+            [cmr.umm.echo10.related-url :as rurl]))
+
+(defn- call-related-url-convert
+  "Call the private function that will build related url types"
+  [related-url-type related-url-sub-type]
+  (#'rurl/related-url->online-resource {:type related-url-type
+                                        :sub-type related-url-sub-type}))
+
+(deftest validate-xml
+  (testing
+    "Validate that the related-url->online-resource function correctly returns
+     subtypes for the DMR use cases"
+    (let [dmr-case (call-related-url-convert "EXTENDED METADATA" "DMR++")
```

Comment:
This test can be cleaned up with `are2` and there is no need for the `call-related-url-convert` function.

Reply:
I thought that we were supposed to use are3 and not are2?

Reply:
yes, are3. I lost track.

Reply:
I assumed are3. Done.